packages/mds-provider/package.json

```diff
@@ -14,8 +14,8 @@
     "test": "yarn test:prettier && yarn test:eslint && yarn test:unit",
     "test:eslint": "eslint --ignore-path ../../.gitignore '**/*.ts'",
     "test:prettier": "prettier --write --check --ignore-path ../../.gitignore '**/*.ts'",
-    "test:unit": "nyc --check-coverage --exclude tests --extension .ts --lines 80 --reporter=text --reporter=html ts-mocha --require source-map-support/register --recursive tests/**/*.ts",
-    "watch": "nodemon --watch '../../packages' --ext 'ts' --ignore '*.d.ts' --exec yarn watch:exec",
+    "test:unit": "nyc --check-coverage --exclude tests --extension .ts --lines 75 --reporter=text --reporter=html ts-mocha --require source-map-support/register --recursive tests/**/*.ts",
```

Comment:
I don't really like the idea of reducing test coverage % significantly with the addition of new features... We should either write new tests which have more coverage, or add istanbul annotations if the code can't be reasonably tested.

Reply:
I did add some new tests and increased the % ... I will update this to `--lines 85`